cmd/modern/main.go

```diff
@@ -95,7 +95,7 @@ func initializeCallback() {
 // nil.  Pass (inject) checkErr into all dependencies (internal helpers etc.) as an
 // errorHandler.
 //
-// To aid debugging issues, if the logging level is > 2 (e.g. -v 3 or -v 4), we
```

Comment:
--verbosity is good for me

Q - what's the standard for when switches use an = sign?

--verbosity=4
or
--verbosity 4 ?

Some apps seem to have a pattern that the "short" flag doesn't use an = but the long one does

Reply:
I think we need some short cut, because a typical support scenario is going to be, please run the command with  "-v 4", and "--verbosity 4" doesn't roll off the tongue.

Reply:
Entered issue for this:

[https://github.com/microsoft/go-sqlcmd/issues/211](https://github.com/microsoft/go-sqlcmd/issues/211)